bucket/multimc.json

```diff
@@ -34,6 +34,6 @@
         "github": "https://github.com/MultiMC/MultiMC5"
     },
     "autoupdate": {
-        "url": "https://github.com/MultiMC/MultiMC5/releases/download/$version/mmc-stable-win32.zip"
+        "url": "https://files.multimc.org/downloads/mmc-stable-win32.zip"
     }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Whats the difference is it just a line ending?